diff --git a/src/osal_dynamiclib_unix.c b/src/osal_dynamiclib_unix.c
index f8ffcdd..da40185 100644
--- a/src/osal_dynamiclib_unix.c
+++ b/src/osal_dynamiclib_unix.c
@@ -33,7 +33,7 @@ m64p_error osal_dynlib_open(m64p_dynlib_handle *pLibHandle, const char *pccLibra
     if (pLibHandle == NULL || pccLibraryPath == NULL)
         return M64ERR_INPUT_ASSERT;
 
-    *pLibHandle = dlopen(pccLibraryPath, RTLD_NOW);
+    *pLibHandle = dlopen(pccLibraryPath, RTLD_NOW | RTLD_GLOBAL);
 
     if (*pLibHandle == NULL)
     {
