diff --git forkSrcPrefix/Makefile.libretro forkDstPrefix/Makefile.libretro
index 1842538..ef2b1f5 100644
--- forkSrcPrefix/Makefile.libretro
+++ forkDstPrefix/Makefile.libretro
@@ -386,6 +386,18 @@ else ifeq ($(platform), gcw0)
    CFLAGS += -fomit-frame-pointer -ffast-math -march=mips32 -mtune=mips32r2 -mhard-float
    CXXFLAGS += $(CFLAGS)
 
+# NxRedux debug
+else ifeq ($(platform), desktop)
+	TARGET := $(TARGET_NAME)_libretro.so
+	CC = $(CROSS_COMPILE)gcc
+	CXX = $(CROSS_COMPILE)g++
+	AR = $(CROSS_COMPILE)ar
+	SHARED := -shared -W
+	LDFLAGS += -fPIC -flto
+	PLATFORM_DEFINES := -DCC_RESAMPLER -DCC_RESAMPLER_NO_HIGHPASS
+	CFLAGS += -fomit-frame-pointer -ffast-math -fPIC -flto
+	CXXFLAGS += $(CFLAGS)
+
 # RETROFW
 else ifeq ($(platform), retrofw)
    TARGET := $(TARGET_NAME)_libretro.so
