diff --git a/Makefile b/Makefile
index 5f6cabc..67968d0 100644
--- a/Makefile
+++ b/Makefile
@@ -83,6 +83,19 @@ else ifeq ($(platform),osx)
 	LDFLAGS += $(ARCHFLAGS)
 
 
+# NxRedux desktop (macOS/Linux host build)
+else ifeq ($(platform), desktop)
+	TARGET := $(TARGET_NAME)_libretro.so
+	CC = $(CROSS_COMPILE)gcc
+	CXX = $(CROSS_COMPILE)g++
+	AR = $(CROSS_COMPILE)ar
+	fpic := -fPIC
+	ifeq ($(shell uname -s), Darwin)
+		SHARED := -shared
+	else
+		SHARED := -shared -Wl,-version-script=$(LIBRETRO_DIR)/link.T -Wl,-no-undefined
+	endif
+
 # iOS
 else ifneq (,$(findstring ios,$(platform)))
 
